src/storage-client/src/types/sources.proto

```diff
@@ -200,11 +200,12 @@ message ProtoPostgresSourceConnection {
     mz_repr.global_id.ProtoGlobalId connection_id = 6;
     mz_storage_client.types.connections.ProtoPostgresConnection connection = 1;
     string publication = 2;
-    ProtoPostgresSourceDetails details = 4;
```

Comment:
@aalexandrov has requested that he is notified about any protobuf changes, so I am pinging him here.

Reply:
This is just renaming the struct, thanks.